test/functional/spec-runner/index.js

```diff
@@ -289,6 +291,10 @@ function runTestSuiteTest(configuration, spec, context) {
     if (IGNORED_COMMANDS.has(event.commandName)) {
       return;
     }
+    // TODO: This should be removed if NODE-2471 accomidates it
+    if (event.command.thisPingShouldBeIgnored) {
```

Comment:
it seems like a more general solution would be to just drop the first ping if credentials are provided?